common/v2/features/SendAssets/types.ts

```diff
@@ -1,8 +1,12 @@
-import { IAsset } from 'v2/types';
-import { Asset, assetMethod } from 'v2/services/Asset/types';
-import { ExtendedAccount as IExtendedAccount, Network } from 'v2/services';
-import { WalletName } from 'v2/config/data';
-import { GasEstimates } from 'v2/api/gas';
+import {
+  Asset,
+  IAsset,
+  TAssetType,
+  ExtendedAccount as IExtendedAccount,
```

Comment:
See comment somewhere above here.

Reply:
A believe it's just legacy and didn't want to change the name.
I think we should get rid of the id of Extended<DataType> which just ads an uuid to the type and instead have all inherit from an object that expects an uuid.